### PR TITLE
create Main UI

### PR DESCRIPTION
**Pull request by NVManh246**
https://edu-redmine.framgia.vn/issues/1251

### CODE REVIEW

**Comment by NVManh246** [comment]
ready

File: app/src/main/res/layout/fragment_favorite.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="3"
+        android:textSize="40dp"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/fragment_favorite.xml]: 10 11 hardcode

File: app/src/main/res/layout/fragment_genres.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="2"
+        android:textSize="40dp"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/fragment_genres.xml]: 10 11 hardcode

File: app/src/main/res/layout/fragment_home.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        xmlns:app="http://schemas.android.com/apk/res-auto">
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="1"
+            android:textSize="40dp"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/fragment_home.xml]: 12 13

**Pull request merged**